database/query/provider_github_app_installations.sql

```diff
@@ -1,6 +1,9 @@
 -- name: GetInstallationIDByProviderID :one
 SELECT * FROM provider_github_app_installations WHERE provider_id = $1;
 
+-- name: GetInstallationIDByAppID :one
+SELECT * FROM provider_github_app_installations WHERE app_installation_id = $1;
```

Comment:
Do we have an index for this?

Reply:
we don't. I'll do that along with the int64 conversion.

Reply:
oops, sorry, actually we do - it's a primary key so it's indexed by default (found this out while writing the conversion to int64)